client/options/index.ts

```diff
@@ -146,9 +147,23 @@ export function initOptions() {
 		new OptionModel(id, specs[id])
 	}
 
+	// Manually change bgVideo select as it is dynamically generated
+	const bgElement = document.getElementById("bgVideo")
+
+	for (let val of bgElement.getElementsByTagName("option")) {
```

Comment:
`bgElement.innerHTML = ""` is cleaner.

Reply:
I think I remember doing that before, but decided not to for some reason.
I'll try again, maybe there was another issue.